src/support/bits.cpp

```diff
@@ -42,7 +42,7 @@ template<> int PopCount<uint32_t>(uint32_t v) {
 }
 
 template<> int PopCount<uint64_t>(uint64_t v) {
-  return PopCount((uint32_t)v) + PopCount((uint32_t)(v >> 32));
+  return PopCount((uint32_t)v) + (v >> 32 ? PopCount((uint32_t)(v >> 32)) : 0);
```

Comment:
What is the motivation for this change? This seems slightly harder to read than before.

Reply:
Main motivation is consistency with CountLeadingZeros for 64-bit and potentially speedup calculation when high part of 64-bit is zero

Reply:
I think the consistency with CountLeadingZeroes is a non-goal here because they are fundamentally different optimizations. If we could rewrite CountLeadingZeroes to match PopCount, I think we would want to do that for improved readability.

If we're going to sacrifice some readability for performance, it would be good to see that the performance difference is measurable rather than hypothetical.

This PR LGTM other than this point, so it might be nice to split this out and land the rest.

Reply:
Alright. I switched to prev implementation for PopCount<uint64_t>